usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+void makeRequest() async{
```

Comment:
There should be a space between `async` and `{`.

Can you use VSC's formatting tool? Right click, "format file" will fix spacing and indentation for you.

Reply:
See this [slide](https://docs.google.com/presentation/d/1tybmc0YPWrdmRy72DM4TAvJq9HPVocZgZB-k9tEdNRo/edit#slide=id.g855f226bc4_9_17).

Reply:
I will now use "format file" function to fix spacing and indentation.